content/src/content/jcr_root/apps/core/wcm/components/embed/v1/embed/embeddable/youtube/youtube.html

```diff
@@ -14,27 +14,14 @@
   ~ limitations under the License.
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
 
-<sly data-sly-use.youTube="com.adobe.cq.wcm.core.components.models.embeddable.YouTube">
+<sly data-sly-use.youTube="com.adobe.cq.wcm.core.components.models.embeddable.YouTube"
+     data-sly-use.fixedTemplate="youtubefixed.html"
+     data-sly-use.responsiveTemplate="youtuberesponsive.html">
   <sly data-sly-test="${!youtube.empty}">
-    <iframe data-sly-test="${!youtube.layout || youtube.layout=='fixed'}"
-            data-cmp-data-layer="${youtube.data.json}"
-            width="${youTube.iFrameWidth || '100%' @ context='scriptString'}"
-            height="${youTube.iFrameHeight || 390 @ context='scriptString'}"
-            src="${youTube.iFrameSrc}"
-            frameborder="0"
-            allowfullscreen
-            allow="autoplay; fullscreen"
-            aria-label="${'YouTube Video' @ i18n}"></iframe>
-    <div data-sly-test="${youtube.layout=='responsive'}"
-         class="cmp-embed__youtube"
-         style="padding-bottom: ${youTube.iFrameAspectRatio || 56.25  @ context='scriptString'}%">
-      <iframe class="cmp-embed__youtube-iframe"
-              data-cmp-data-layer="${youtube.data.json}"
-              src="${youTube.iFrameSrc}"
-              frameborder="0"
-              allowfullscreen
-              allow="autoplay; fullscreen"
-              aria-label="${'YouTube Video' @ i18n}"></iframe>
-    </div>
+    <sly data-sly-test="${!youtube.layout || youtube.layout!='responsive'}"
+         data-sly-call="${youtubefixed.youtube @ youtube=youtube}"></sly>
+
+    <sly data-sly-test="${youtube.layout=='responsive'}"
+         data-sly-call="${youtuberesponsive.youtube @ youtube=youtube}"></sly>
```

Comment:
same here, i guess this should be:
`data-sly-call="${responsiveTemplate.youtube @ youtube=youtube}"`